Jenkinsfile

```diff
@@ -186,6 +186,12 @@ ansiColor('xterm') {
             }
           }
 
+          stage('Check for ES5 Compatibility') {
+            sh'''
+              node_modules/.bin/es-check es5 packages/node_modules/@ciscospark/**/dist/*.js
```

Comment:
This should also check the `/esm` and `/cjs` folders. The `esm` folder is what webpack and other module loaders will load and aren't usually configured to transpile code. 

We should also add this step to the [`publish:components` helper](https://github.com/webex/react-ciscospark/blob/cd79c7d54bcd5728d9d68e5401d8af1fca67ae1c/Jenkinsfile#L246) to verify those esm and cjs folders are good

Reply:
will need to talk more about this tomorrow.